packages/@react-types/inlinealert/src/index.d.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {DOMProps, StyleProps} from '@react-types/shared';
+import {ReactNode} from 'react';
+
+
+export interface SpectrumInLineAlertProps extends DOMProps, StyleProps {
+  variant?: 'neutral' | 'info' | 'positive' | 'notice' | 'negative',
```

Comment:
need to add comments to these for the props table in the docs
```suggestion
   /**
   * The [visual style](https://spectrum.adobe.com/page/in-line-alert/#Options) of the In Line Alert.
   * @default 'neutral'
   */
  variant?: 'neutral' | 'info' | 'positive' | 'notice' | 'negative',
```